source/codegen/generate_shared_service_files.py

```diff
@@ -7,7 +7,7 @@ def generate_register_all_services(metadata_dir: Path, output_dir: Path) -> None
   driver_modules = [
     load_metadata(p)
     for p in sorted(metadata_dir.iterdir())
-    if p.is_dir() and not "fake" in p.name
+    if p.is_dir() and not "fake" in p.name and not p.name.startswith(".")
```

Comment:
Curious: what is this change about?

Reply:
Oops, I didn't mean for this to go into this change. I'll revert it.

I made the change, though, because I had a .mypy_cache folder in the same path.